mount.go

```diff
@@ -29,49 +29,29 @@ type Server interface {
 	ServeOps(*Connection)
 }
 
-// Mount attempts to mount a file system on the given directory, using the
-// supplied Server to serve connection requests. It blocks until the file
-// system is successfully mounted.
-func Mount(
-	dir string,
+// FuseServe attempts to start the supplied fuse server with the file discriptor
+// to poll against. It blocks until the file system is successfully mounted.
+func FuseServe(
```

Comment:
Could you please describe in the documentation for this and the function below when you would want to use them versus just using Mount? Having three entry points is confusing; we should at least have the more complex ones give a pointer to the simpler ones.

Reply:
I'll add a comment. Thanks.